### ISSUE TITLE
Need NDNFit Android application bootstrap function

### ISSUE THREAD

**Issue by zhtaoxiang** [zhehaowang/ndnfit#5]
1. Currently, if a user restarts the Android application, the application can continue to work correctly.
2. In the future, if a user reinstalls the Android application, the application should also continue to work correctly. A potential solution is the Android application fetches its latest status from the DSU.

**Comment by zhehaowang** [comment]
You mean the app keeps the user's identity choices (so essentially the publishing prefix, too) even after uninstallation and wiping data, right?

Yes we could have the application fetch this from the DSU (which may store generic "user decisions", like "use this identity for this app on this device", "I granted that person access of my walking data", etc).

Side note, upon reinstallation, a different application identity will be created, and will be signed by the local ID manager, if the local ID manager has the identity indicated by DSU.

**Comment by zhtaoxiang** [comment]
Yes, I mean a user should be able to (if the user want) resume all the latest status (what you mentioned should be part of the status, but there may be more, we need a discussion about what else we need) when he/she uninstalls the application.

**Comment by zhehaowang** [comment]
Yes; let's keep this open till discussed/resolved.
